src/functionalTest/resources/responses/response-cic-submit-case-no-notification.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this can be replaced with the shortened version using
```
assertThatJson(response.asString())
            .inPath("$.confirmation_header")
            .isEqualTo("# Application Received %n##");
```